Part of request but unworking at the moment

Edith-D opened the pull request.

Edith-D committed:
Part of request but unworking at the moment

ivyshi98 commented on Shared/Communicator.cs:
```diff
@@ -45,10 +45,9 @@ private async Task<Object> OnQuery(IRequest request)
-            return null;
+            IResponse response = new Response(true, request.CorrelationId, null, tmp.ToString(), request.Sender, request.Timestamp);
+            await connection.InvokeAsync<Response>("RespondQueryAsync", response);
+            return Task.FromResult(response);
```
Also did not have a return for the method

Edith-D committed:
Fixe the Request and response
Modified in the Hub to set in dictionary
Add in CorEstafette startup a maximum user connection liit to 5

Edith-D committed:
Merge branch 'master' into Request
